pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -129,6 +129,7 @@ def get_unit_data(self, unit):
 
 class Command(BaseCommand):
     help = "Load Translation Memory with translations"
+    args = "[files]"
```

Comment:
`files` is enclosed in `[]` to tell that this is optional.

Not sure what you mean with more than one usage.